statistics/table.go

```diff
@@ -563,19 +563,57 @@ func outOfRangeEQSelectivity(ndv, modifyRows, totalRows int64) float64 {
 	return selectivity
 }
 
+// crossValidationSelectivity gets the selectivity of multi-column equal conditions by cross validation.
+func (coll *HistColl) crossValidationSelectivity(sc *stmtctx.StatementContext, idx *Index, usedColsLen int, idxPointRange *ranger.Range) (float64, float64, error) {
+	minRowCount := math.MaxFloat64
+	cols := coll.Idx2ColumnIDs[idx.ID]
+	crossValidationCount := 1.0
+	totalRowCount := float64(idx.TotalRowCount())
+	for i, colID := range cols {
+		if i >= usedColsLen {
+			break
+		}
+		if col, ok := coll.Columns[colID]; ok {
+			lowExclude := idxPointRange.LowExclude
+			highExclue := idxPointRange.HighExclude
+			if lowExclude != highExclue && i < usedColsLen {
+				lowExclude = false
+				highExclue = false
+			}
```

Comment:
What does this if branch mean?

Reply:
Consider this case.
```
create table t(a int, b int, c int, primary key(a,b,c));
insert into t values(1,1,1),(2,2,3);
explain select * from t where (a,b) in ((1,1),(2,2)) and c > 2;
```
We get range: `(1 1 2,1 1 +inf], (2 2 2,2 2 +inf]`

For column `a`, we will get range: `(1, 1], (2, 2]`, but `rowCount, err := col.GetColumnRowCount(sc, []*ranger.Range{&rang}, coll.ModifyCount, col.IsHandle)`  with rang = `(2, 2]` will return 0. And the result of explain will output `estRow 0.0`. So I change it to `[2, 2]`.

Reply:
Please add this case as a comment into code to explain the rationale.